spark/src/test/scala/org/apache/spark/sql/delta/typewidening/TypeWideningInsertSchemaEvolutionSuite.scala

```diff
@@ -310,10 +309,11 @@ trait TypeWideningInsertSchemaEvolutionTests
       Seq("""{ "key": 1, "m": { "a": { "x": 3, "y": 4 } } }""")),
     expectedResult = ExpectedResult.Success(new StructType()
       .add("key", IntegerType)
-      // Type evolution wasn't applied in the map.
+      // Type evolution now works for map.
       .add("m", MapType(StringType, new StructType()
         .add("x", IntegerType)
-        .add("y", ShortType)))),
+        .add("y", IntegerType, nullable = true,
+          metadata = typeWideningMetadata(version = 1, from = ShortType, to = IntegerType))))),
```

Comment:
You can actually remove this test case altogether and update the one below to remove `includeInserts` so that it will cover all insert types.